docs/examples/form/Checkbox/README.md

```diff
@@ -1 +1 @@
-undefined
\ No newline at end of file
+This component is used to provide a boolean state for a particular field in a form. You can optionally proide content to the checkbox, placing it in the center.
```

Comment:
proide is mispelled -- ** provide **

Reply:
Also, that second sentence is confusing to me when I read it. Perhaps the example will make elucidate further? On its own, not clear though

Reply:
I will definitely add an example of what that second sentence is meant to mean, although it probably could be worded differently. I was attempting to convey that this same element can be used for an empty checkbox as well as this component: 

<img width="116" alt="screen shot 2018-04-23 at 10 58 23 pm" src="https://user-images.githubusercontent.com/916317/39168783-07a29ce8-474a-11e8-99a8-d413d256917f.png">